documentation/puppet/v3/environment_classes.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
I used `JSON::Validator` in the Ruby `json-schema` gem (https://github.com/ruby-json-schema/json-schema) to validate this schema against the response data from some sample queries that I made to the `environment_classes` endpoint.  That said, it wouldn't surprise me if I have some errors in this doc given my n00b-level familiarity with JSON schema.  If there's a better tool to use to validate a JSON schema, I'd be happy to use that.